third_party/move/move-compiler-v2/tests/reference-safety/v1-tests/dereference_field.exp

```diff
@@ -1,9 +0,0 @@
-
-Diagnostics:
-error: cannot copy mutable reference in local `s` which is still borrowed
```

Comment:
These test cases are terrible.  It's easy to argue that the error shouldn't be there, because nothing read through a reference is every used, and nothing is modified through a `&mut`.  Do we have a user-comprehensible description of the borrowing rules somewhere?  Otherwise, whether there is an error or not seems pretty arbitrary.  I can see our users getting frustrated by this.